### PR TITLE
Kotlin offset in

### LINKED ISSUE #3816: [Term Entry] Kotlin Datetime .offsetIn()
### Reason/inspiration (optional)

We would like a new term entry on the `.offsetIn()` method for Kotlin: [Datetime](https://www.codecademy.com/resources/docs/kotlin/datetime). The entry should go in a new file under `docs/content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by vrun1208**
### Description

closes #3816 
Added a new term entry for kotlin dateTime .offsetIn() method

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  val offset = currentTime.offsetIn(timeZone)
+
+  println("Offset from UTC in Kolkata is $offset minutes.")
+}
+```
+
+The code given above provides the offset for the current time in `Asia/Kolkata` time zone.
+
+**Note:** the string `Asia/Kolkata` above is a zoneId (returns the time zone identified by the provided zoneId). These Id's are usually in the form of `area/city`.
```
Comment (avdhoottt) [inline content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md:44]:
```suggestion
> **Note:** The string `Asia/Kolkata` above is a zoneId (returns the time zone identified by the provided zoneId). These IDs are usually in the form of `area/city`.
```
You can check out the appropriate way to create a note [here](https://github.com/vrun1208/docs/blob/kotlin-periodUntil/documentation/style-guide.md). By the way, it's great that you thought of writing it.

File: content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import kotlinx.datetime.TimeZone
+
+fun main() {
+  val currentTime = Instant.now()
+  val timeZone = TimeZone.of("Asia/Kolkata")
+
+  val offset = currentTime.offsetIn(timeZone)
+
+  println("Offset from UTC in Kolkata is $offset minutes.")
+}
```
Comment (avdhoottt) [inline content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md:39]:
```suggestion
import kotlinx.datetime.TimeZone

fun main() {
  val currentTime = Instant.now()
  val timeZone = TimeZone.of("Asia/Kolkata")

  val offset = currentTime.offsetIn(timeZone)

  println("Offset from UTC in Kolkata is $offset minutes.")
}
```
The now() function was deprecated in Kotlin Datetime 0.3.0 so use any alternative function.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@vrun1208 Please make these changes and you have committed two files. Don't commit the periodUntil.md in this entry.

File: content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md (hunk 2 above)
Comment (vrun1208) [reply content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md:39]: Sure I'll do it soon! And what to do with periodUntil.md file now it somehow get commited by me

Comment (avdhoottt) [reply content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md:39]: @vrun1208 Delete periodUntil.md file from the branch then commit and push the changes.

File: content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **offsetIn()** function is used to obtain the time zone offset at a specific point in time. It provides the difference between the local time and UTC for a given moment.
```
Comment (avdhoottt) [inline content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md:15]:
```suggestion
The **`.offsetIn()`** function is used to obtain the time zone offset at a specific point in time. It provides the difference between the local time and UTC for a given moment.
```

File: content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md (hunk 2 above)
Comment (vrun1208) [reply content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md]: @avdhoottt, Clock.System.now() would be an alternative for now() by importing kotlinx.datetime.Clock

Comment (avdhoottt) [reply content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md]: @vrun1208 You can use it then..

File: content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import kotlinx.datetime.Clock
+
+fun main() {
+  val currentTime = Clock.System.now()
+  val timeZone = TimeZone.of("Asia/Kolkata")
+
+  val offset = currentTime.offsetIn(timeZone)
+
+  println("Offset from UTC in Kolkata is $offset.")
+}
```
Comment (avdhoottt) [inline content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md:40]:
```suggestion
import kotlinx.datetime.TimeZone
import kotlinx.datetime.Clock
import kotlinx.datetime.offsetIn

fun main() {
  val currentTime = Clock.System.now()
  val timeZone = TimeZone.of("Asia/Kolkata")

  val offset = currentTime.offsetIn(timeZone)

  println("Offset from UTC in Kolkata is $offset.")
}
```
You need to import the function offsetIn too.

**Review by avdhoottt — APPROVED** [review]
PR is good to go for the second review.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good to merge.

**Pull request merged**